homeassistant/components/waterfurnace.py

```diff
@@ -0,0 +1,91 @@
+"""
+Support for Waterfurnace component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/waterfurnace/
+"""
+from datetime import timedelta
+import logging
+import time
+import threading
+import voluptuous as vol
```

Comment:
Add a blank line between import groups standard library, 3rd party and homeassistant.

Reply:
ack